src/composables/useUserPreferences.js

```diff
@@ -0,0 +1,80 @@
+import { gql } from 'graphql-tag';
+
+export default function useUserPreferences(apollo) {
+	const createUserPreferences = () => {
+		try {
+			const createUserPreferencesMutation = gql`
+				mutation createUserPreferences($preferences: String) {
+					my {
+						createUserPreferences(userPreferences: {preferences: $preferences}) {
+							id
+							preferences
+						}
+					}
+				}
+			`;
+
+			return apollo.mutate({
+				mutation: createUserPreferencesMutation,
+				variables: {
+					preferences: '',
+				},
+			});
+		} catch (error) {
+			throw new Error(error);
```

Comment:
We don't want to re-throw the exceptions in these 3 methods. We want to instead use `logReadQueryError`